addons/interaction/functions/fnc_start_desk_close.sqf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Starts process of closing a desk
+ *
+ * Arguments:
+ * 0: Desk <OBJECT>
+ *
+ * Return Value:
+ * None
+*/
+
+params ['_desk'];
+
+[
+	15, // time in seconds
+	_desk, // arguments for Finish + fail
+	{ // on finish
+		[_this] call AE3_interaction_fnc_desk_close;
```

Comment:
`_this` is not correct, as it contains more then the given arguments. You should change this to `_this select 0`, I think. Or take a look into several other implementations of `progressBar`, for example here: https://github.com/y0014984/Advanced-Equipment/blob/dev/addons/power/functions/fnc_turnOffGeneratorAction.sqf